src/app/shared/sidebar/sidebar-dropdown.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="setting-option mb-3 p-3">
-  <h4><label for="{{id}}">{{label | translate}}</label></h4>
+  <label class="h5" for="{{id}}">{{label | translate}}</label>
```

Comment:
I don't understand here why we had to change an `<h4>` into a `<label>`?  Could you explain the purpose of this change?

Reply:
1 accessibility violation was detected